### PR TITLE
docs: Update README.md

### PR DESCRIPTION
**Pull request by danielabrozzoni**
### Description

- Fixes the release timeline
- Adds an explanation for the crates under `example-crates`

#### All Submissions:

* [x] I've signed all my commits
* [x] I followed the [contribution guidelines](https://github.com/bitcoindevkit/bdk/blob/master/CONTRIBUTING.md)
* [x] I ran `cargo fmt` and `cargo clippy` before committing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,9 +45,14 @@ The project is split up into several crates in the `/crates` directory:
 - [`esplora`](./crates/esplora): Extends the [`esplora-client`] crate with methods to fetch chain data from an esplora HTTP server in the form that [`bdk_chain`] and `Wallet` can consume.
 - [`electrum`](./crates/electrum): Extends the [`electrum-client`] crate with methods to fetch chain data from an electrum server in the form that [`bdk_chain`] and `Wallet` can consume.
 
-Fully working examples of how to use these components are in `/example-crates`
+Fully working examples of how to use these components are in `/example-crates`:
+- [`example_cli`](./example-crates/example_cli): Library used by the `example_*` crates. Provides utilities for syncing, showing the balance, generating addresses and creating transactions without using the bdk `Wallet`.
+- [`example_`](./example-crates/example_electrum): A command line Bitcoin wallet application built on top of `example_cli` and the `electrum` crate. It shows the power of the bdk tools (`chain` + `file_store` + `electrum`), without depending on the main `bdk` library.
```
Comment (vladimirfomene) [inline README.md]: I think you meant to write [`example_electrum`]

**Comment by vladimirfomene** [comment]
I also found myself confused about the difference between `example-crates/example_*` and `example-crates/wallet_*`. This will be very useful for our users

File: README.md [hunk 2]
```diff
@@ -45,9 +45,14 @@ The project is split up into several crates in the `/crates` directory:
...
+- [`example_electrum`](./example-crates/example_electrum): A command line Bitcoin wallet application built on top of `example_cli` and the `electrum` crate. It shows the power of the bdk tools (`chain` + `file_store` + `electrum`), without depending on the main `bdk` library.
+- [`wallet_esplora`](./example-crates/wallet_esplora): Uses the `Wallet` to sync and spend using Esplora.
+- [`wallet_esplora_async`](./example-crates/wallet_esplora_async): Uses the `Wallet` to sync and spend using the Esplora asynchronous interface.
+- [`wallet_electrum`](./example-crates/wallet_electrum): Uses the `Wallet` to sync and spend using Electrum.
 
-[`bdk_core_staging`]: https://github.com/LLFourn/bdk_core_staging
+[`BDK 1.0 project page`]: https://github.com/orgs/bitcoindevkit/projects/14
 [`rust-miniscript`]: https://github.com/rust-bitcoin/rust-miniscript
 [`rust-bitcoin`]: https://github.com/rust-bitcoin/rust-bitcoin
 [`esplora-client`]: https://docs.rs/esplora-client/0.3.0/esplora_client/
```
Comment (notmandatory) [inline README.md]: Overall like the changes, just needs some link fixes. Better not to hard code the versions so we don't need to keep updating them. Also need to add `bdk_chain`.
```suggestion
[`esplora-client`]: https://docs.rs/esplora-client/
[`electrum-client`]: https://docs.rs/electrum-client/
[`bdk_chain`]: https://docs.rs/bdk-chain/
```

File: README.md [hunk 3]
```diff
@@ -45,9 +45,14 @@ The project is split up into several crates in the `/crates` directory:
 - [`esplora`](./crates/esplora): Extends the [`esplora-client`] crate with methods to fetch chain data from an esplora HTTP server in the form that [`bdk_chain`] and `Wallet` can consume.
 - [`electrum`](./crates/electrum): Extends the [`electrum-client`] crate with methods to fetch chain data from an electrum server in the form that [`bdk_chain`] and `Wallet` can consume.
 
-Fully working examples of how to use these components are in `/example-crates`
+Fully working examples of how to use these components are in `/example-crates`:
+- [`example_cli`](./example-crates/example_cli): Library used by the `example_*` crates. Provides utilities for syncing, showing the balance, generating addresses and creating transactions without using the bdk `Wallet`.
+- [`example_electrum`](./example-crates/example_electrum): A command line Bitcoin wallet application built on top of `example_cli` and the `electrum` crate. It shows the power of the bdk tools (`chain` + `file_store` + `electrum`), without depending on the main `bdk` library.
+- [`wallet_esplora`](./example-crates/wallet_esplora): Uses the `Wallet` to sync and spend using Esplora.
```
Comment (realeinherjar) [inline README.md]:
```suggestion
- [`wallet_esplora`](./example-crates/wallet_esplora): Uses the `Wallet` to sync and spend using Esplora blocking interface.
```

**Review by realeinherjar — COMMENTED** [review]
I think it would be nice to add the same naming convention in `bdk_esplora`.

**Review by notmandatory — CHANGES_REQUESTED** [review]

**Comment by danielabrozzoni** [comment]
Thanks everyone, updated.

> I think it would be nice to add the same naming convention in bdk_esplora.

@realeinherjar can you clarify? I haven't added `example_esplora` to the list of `example_crates` as it's not merged yet. Or were you referring to something else?

**Comment by realeinherjar** [comment]
> @realeinherjar can you clarify? I haven't added `example_esplora` to the list of `example_crates` as it's not merged yet. Or were you referring to something else?

@danielabrozzoni just the naming conventions for consistency.
At the [`bdk_esplora` docs](https://docs.rs/bdk_esplora/0.3.0/bdk_esplora/) we have this:
image

So why not add the "blocking" here as well?

**Comment by danielabrozzoni** [comment]
Right, I pushed a commit renaming `wallet_esplora` to `wallet_esplora_blocking`

**Comment by danielabrozzoni** [comment]
Rebased, CI is fixed now.

**Review by notmandatory — APPROVED** [review]

**Review by notmandatory — APPROVED** [review]
ACK 2263a58448f5f04a74e076132aea93f21c584bc0

**Review by evanlinjin — APPROVED** [review]
ACK 2263a58448f5f04a74e076132aea93f21c584bc0

**Pull request merged**